sdcm/nemesis/monkey/sla.py

```
@@ -0,0 +1,231 @@
+from test_lib.sla import MAX_ALLOWED_SERVICE_LEVELS, ServiceLevel
+
+
+class SlaMonkeyBase(NemesisBaseClass):
+    """Base class for SLA nemesis with shared validation logic."""
+
+    sla = True
+
+    additional_configs = [
+        "configurations/nemesis/additional_configs/sla_config.yaml",
+        "configurations/auth_cassandra.yaml",
+    ]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)